cmd/analyzer/analyzer.go

```diff
@@ -84,11 +107,32 @@ func parseFlags() cliFlags {
 	if cli.shardPrefixesFlag != "" {
 		cli.shardPrefixes = strings.Split(cli.shardPrefixesFlag, ",")
 	}
+
+	if cli.json {
+		data := &MistConfig{
+			Name:         "LivepeerAnalyzer",
+			Version:      version,
+			Description:  "",
+			FriendlyName: "Livepeer Analyzer",
+			Optional:     []MistOptional{},
+		}
+		fs.VisitAll(func(f *flag.Flag) {
+			data.Optional = append(data.Optional, MistOptional{
+				Name:    f.Name,
+				Help:    f.Usage,
+				Default: f.DefValue,
+			})
+		})
+		b, _ := json.Marshal(data)
+		os.Stdout.Write(b)
```

Comment:
WDYT of moving all this to a helper function like `printMistJsonConfig(version, fs)` or sth like that? Keeping only the `os.Exit` out of the helper so it's clear here that we never continue if we this branch.

Bonus: It could even be a lib under `livepeer-data/pkg`, so we can just reuse it from the other go services (or copy it, but still keeping it isolated). It could hold all the related stuff like the declared `MistConfig` and `MistOptional` data types, so we have a very minimal interface here, passing only the name(s), description, version and `FlagSet`.